nbbuild/build.xml

```diff
@@ -1968,7 +1968,8 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
                 <exclude name="diff/test/unit/src/org/netbeans/modules/diff/builtin/visualizer/data/**" /> <!--test data-->
                 <exclude name="utilities/test/unit/src/org/netbeans/modules/openfile/resources/recent_files/*" /> <!--test data-->
                 <exclude name="**/*.url" /> <!--no degree of creativity-->
-		<exclude name="spi.navigator/test/unit/src/org/netbeans/modules/navigator/resources/lastsel/file.lastsel_mime1" /> <!--empty file-->    
+		            <exclude name="spi.navigator/test/unit/src/org/netbeans/modules/navigator/resources/lastsel/file.lastsel_mime1" /> <!--empty file-->    
```

Comment:
extra tab?

Reply:
ups; person that created spi.navigator-entry used tabs. my editor showed it missaligned and thus i added more spaces. fixed it.